content/Api/Blip.Api.Template.Models/Blip.Api.Template.Models.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
-    <ProjectGuid>sonarqube-guid-models</ProjectGuid>
+    <ProjectGuid>{46EC828B-835E-4475-B0ED-F1EFC0DBCB51}</ProjectGuid>
```

Comment:
whenever you build the project, or open in VS even, the placeholder gets replaced by the generated GUID. Revert back to the placeholder